recipes-browser/wpewebkit/wpewebkit.inc

```diff
@@ -19,9 +19,10 @@ CCACHE_DISABLE[unexport] = "1"
 
 TOOLCHAIN = "gcc"
 
-PACKAGECONFIG ??= "fetchapi indexeddb mediasource video webaudio webcrypto woff2 gst_gl remote-inspector openjpeg"
+PACKAGECONFIG ??= "fetchapi indexeddb mediasource video webaudio webcrypto woff2 gst_gl remote-inspector openjpeg atk"
 
 # WPE features
+PACKAGECONFIG[atk] = "-DENABLE_ACCESSIBILITY=ON,-DENABLE_ACCESSIBILITY=OFF,atk"
```

Comment:
Can it be named accessibility instead of atk? I think the at-spi2-core dep is also missing.

Reply:
https://git.yoctoproject.org/cgit/cgit.cgi/poky/commit/?id=67edffa0aeb0830a8c4a88b3643c9654396dfab3

Reply:
> accessibility

+1